core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -19,7 +19,10 @@
 // TODO-LDML: namespace com.keyman.core.ldml {
 /**
  * Constants for the KMXPlus data format
- * These are shared between the data access layer and the compiler
+ * These are shared between the data access layer and the compiler.
+ * Note that the section IDs (section_keys etc.) are 32 bit hex
+ * values that are designed to appear as text when written in little endian
+ * format, so 0x7379656b = 'keys'
```

Comment:
Would it be possible to do this translation dynamically in our typescript code so we can have nice strings here in the source .ts file?

And then the .h file could have e.g.

```
#define LDML_SECTIONID_KEYS 0x7379656B
#define LDML_SECTIONNAME_KEYS "keys"
```

Reply:
that makes sense as an approach.  I'll add a helper function in TS and use it in the generator.

Reply:
done